google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -217,13 +218,47 @@ def _get_bq_execute_params(operator_task_details):
       operator_task_details['bql'] = operator_task_details['query'].sql
       del operator_task_details['query']
 
+    if 'parameters' in operator_task_details:
+      operator_task_details['query_params'] = Pipeline._get_query_parameters(
+        operator_task_details['parameters'])
```

Comment:
You can do "operator_task_details.pop('parameters')" to save the next "del" line. Up to you.

Reply:
Noted. I vaguely recall running into issues with the py35 tests when I had it this way originally. But I forget what it was, and will investigate and sweep up in a subsequent refactoring.